src/components/history/messages/regular/task/Header.js

```diff
@@ -0,0 +1,73 @@
+import React, {PureComponent} from 'react'
+import injectSheet from 'grape-web/lib/jss'
+import {small, normal} from 'grape-theme/dist/fonts'
+import {ellipsis} from 'grape-web/lib/jss-utils/mixins'
+import {grayBlueLighter} from 'grape-theme/dist/base-colors'
+import Icon from 'grape-web/lib/svg-icons/Icon'
+
+import IconButton from './IconButton'
+
+@injectSheet({
+  header: {
+    position: 'relative',
+    background: grayBlueLighter,
+    padding: 10,
+    display: 'flex',
+    alignItems: 'flex-start'
+  },
```

Comment:
Some projects put them at the end because they are "noisy". e.g. https://git.io/vyLB9

I think that it is good to have them in the same file so that one doesn't have to create and/or leave the component's file but I'd place them at the bottom.